static/app/views/dashboards/widgets/bigNumberWidget/thresholdsIndicator.tsx

```diff
@@ -5,10 +5,18 @@ import type {Polarity} from 'sentry/components/percentChange';
 
 import {normalizeUnit} from '../../utils';
 import {ThresholdsHoverWrapper} from '../../widgetBuilder/buildSteps/thresholdsStep/thresholdsHoverWrapper';
-import type {Thresholds} from '../common/types';
+import type {ThresholdsConfig} from '../../widgetBuilder/buildSteps/thresholdsStep/thresholdsStep';
+
+type ValidThresholds = {
+  max_values: {
+    max1: number;
+    max2: number;
+  };
+  unit?: string;
+};
```

Comment:
```suggestion
type ValidThresholds = ThresholdsConfig & {
  max_values: Required<ThresholdsConfig['max_values']>;
};
```
nit: so i'm not really sure if this is any cleaner, but maybe something like this will show the relationship to `ThresholdsConfig` a bit more clearly?